.github/workflows/ci.yml

```diff
@@ -10,25 +10,29 @@ jobs:
       fail-fast: false
       matrix:
         php-version: ['8.1', '8.3']
+        dependencies: ['highest']
+        include:
+            - php-version: '8.1'
+              dependencies: 'lowest'
+            - php-version: '8.3'
```

Comment:
Isnt that just replicating the above?

Reply:
I added this to be able to define lowest and highest stability